dev-libs/libcdio/patches/libcdio-2.0.0.patchset

```diff
@@ -0,0 +1,51 @@
+From c709eabd76da960651c9d5d73b80713d8bde1947 Mon Sep 17 00:00:00 2001
+From: Jerome Duval <jerome.duval@gmail.com>
+Date: Sat, 19 Apr 2014 21:36:18 +0000
+Subject: Fix gcc2 build.
+
+
+diff --git a/configure.ac b/configure.ac
+index f8f7f7d..dc352ab 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -260,7 +260,7 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([
+ dnl empty_array_size
+ AC_MSG_CHECKING([how to create empty arrays])
+ 
+-empty_array_size="xxxx"
++empty_array_size="xxx"
```

Comment:
For some reason this doesn't work when trying to build vcdimager latest version ....
/packages/libcdio-2.0.0-1/.self/develop/headers/cdio/rock.h:226: `xxxx' undeclared here (not in a function)

Reply:
Clean your work folder and try again? If it still doesn't work then can you post the entire build log?